CommandLine/libEGM/gmx.cpp

```diff
@@ -29,15 +29,16 @@
 #include <vector>
 
 using CppType = google::protobuf::FieldDescriptor::CppType;
+using LookupMap = std::unordered_map<std::string, std::unordered_map<int, std::string>>;
```

Comment:
Totally unnecessary, it already exists.
https://github.com/enigma-dev/enigma-dev/blob/b0375a3f7f25897f680e9d47dcb517406f82b15c/CommandLine/libEGM/gmx.cpp#L82

Reply:
the string in that map is the resource type not the name

Reply:
Oh, it's the base id for that resource type. That's how I used it in #1989 to fix instance ids.

Reply:
it could probably stand to be renamed as I was confused to its purpose as well

Reply:
Just call it typeBaseId since it's the last id used for each resource. Or call it `maxId` or `maxTypeId` or something.

Reply:
I wasn't planning on doing it here but if you want to shoehorn it in I have allow edits from maintainers on